frontend/app/src/components/display/AccountDisplay.vue

```diff
@@ -46,47 +46,38 @@ const { t } = useI18n();
 </script>
 
 <template>
-  <VTooltip top open-delay="400" :disabled="!truncate">
-    <template #activator="{ on }">
-      <VRow align="center" no-gutters class="flex-nowrap" v-on="on">
-        <VCol v-if="!hideChainIcon" cols="auto" class="pr-2">
-          <VAvatar left size="28px" class="mr-0">
-            <ChainIcon
-              v-if="account.chain && account.chain !== 'ALL'"
-              size="24px"
-              :chain="account.chain"
-            />
-            <VTooltip v-else top>
-              <template #activator="{ childOn }">
-                <VIcon v-on="childOn"> mdi-link-variant </VIcon>
-              </template>
-              <span>{{ t('common.multi_chain') }}</span>
-            </VTooltip>
-          </VAvatar>
-        </VCol>
+  <RuiTooltip
+    :popper="{ placement: 'top' }"
+    open-delay="400"
+    :disabled="!truncate"
+    class="flex items-center flex-nowrap"
+  >
+    <template #activator>
+      <div v-if="!hideChainIcon" class="pr-1">
+        <VAvatar left size="28px">
```

Comment:
We don't have a replacement implemented for VAvatar at the moment right?

Reply:
Yeah, not yet